pkg/network/ebpf/c/protocols/kafka/kafka-parsing.h

```diff
@@ -1618,12 +1619,15 @@ static __always_inline bool kafka_process(conn_tuple_t *tup, kafka_info_t *kafka
     kafka_header.correlation_id = bpf_ntohl(kafka_header.correlation_id);
     kafka_header.client_id_size = bpf_ntohs(kafka_header.client_id_size);
 
-    log_debug("kafka: kafka_header.api_key: %d api_version: %d", kafka_header.api_key, kafka_header.api_version);
 
     if (!is_valid_kafka_request_header(&kafka_header)) {
         return false;
     }
```

Comment:
wouldn't this condition drop any unsupport-by-decoding produce/fetch messages?

Reply:
It will, but the next PR will split classification and decoding, it's already tested and approved so I didn't want to move it to this PR